app/assets/stylesheets/welcome.scss

```diff
@@ -45,10 +41,15 @@ input#url_original_url {
 .btn {
   border-radius: 25px;
   background: rgba(209, 85, 233, 0.72);
-  border: #9516c7;
+  outline: none;
+  border: none;
   padding: 6px;
 }
 
+.btn:hover {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces